libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/PasscodeActivity.java

```diff
@@ -203,7 +203,7 @@ public void enableLogout(boolean b) {
     public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
         Log.i("onEditorAction", "view=" + v + " actionId=" + actionId + " event=" + event);
         // Processing the editor action only on key up to avoid sending events like pass code manager unlock twice.
-        if (event != null && event.getAction() == KeyEvent.ACTION_UP) {
```

Comment:
Hitting ```ENTER``` manually triggers 2 events - ```ACTION_DOWN``` and then ```ACTION_UP```. We need to simulate ```ACTION_UP``` in the test. Let's revert the changes to this file.